### PR TITLE
added api testing

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
testing new .travis file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -83,6 +83,9 @@ before_install:
     # Get our own Swagger verifier
     - wget https://raw.githubusercontent.com/mendersoftware/autodocs/master/verify_docs.py
 
+    # Clone integration repo. for api testing
+    - git clone https://${GITHUB_TOKEN}@github.com/mendersoftware/integration.git
```
Comment (maciejmrowiec) [inline .travis.yml]: `GITHUB_TOKEN` will not be set on pull request builds which may mess up those builds a bit.

Why using tokens anyway? that repo is public.

Comment (GregorioDiStefano) [inline .travis.yml]: I think I added before it was public :) thanks

**Comment by GregorioDiStefano** [comment]
@maciejmrowiec is you'd like to take a look? 
not sure who else would be interested, but please just tag anyone

**Comment by maciejmrowiec** [comment]
@kjaskiewiczz @mchalski may be interested

**Comment by GregorioDiStefano** [comment]
ping

**Comment by GregorioDiStefano** [comment]
This is still pending review, can someone take a look?
@maciejmrowiec @kjaskiewiczz @mchalski

**Comment by GregorioDiStefano** [comment]
looks like some recent changes broke the acceptance tests... fixing..

**Comment by marekswiecznik** [comment]
looks good, but there is an error/warning when building locally (and also on travis): `cp: cannot stat '../docs/integrations_api.yml': No such file or directory`

**Pull request merged**